pkg/state/chart_dependency.go

```diff
@@ -108,28 +111,16 @@ func (d *ResolvedDependencies) add(dep ResolvedChartDependency) error {
 	return nil
 }
 
-func (d *ResolvedDependencies) Get(chart, versionConstraint string) (string, error) {
-	if versionConstraint == "" {
-		versionConstraint = "*"
-	}
-
+func (d *ResolvedDependencies) Get(chart, release string) (string, error) {
 	deps, exists := d.deps[chart]
```

Comment:
Can we key `d.deps` by aliases instead of the chart names if we introduce `alias` to the lockfile? That way I think we don't need to compare `dep.Alias` against the release name.

Reply:
yes sure !